guides/v2.3/rest/modules/inventory/link-stocks-sources.md

```diff
@@ -0,0 +1,199 @@
+GET /V1/inventory/stock-source-links
+```
+
+**StockSourceLink parameters**
+
+Name | Description | Type | Requirements
+--- | --- | --- | ---
+`stock_id`	| Stock ID | Integer | Required for POST operations
+`source_code`	| Source code of the link | String | Required for POST operations
+`priority`	| Priority of the link | Integer | Required when linking a stock to a source
+{:style="table-layout:auto;"}
+
+## Link stocks and sources
+
+If you specify a sales channel that is already assigned to another stock, Magento reassigns the sales channel from the previous stock to the current stock.
+
+**Sample usage**
+
+`POST /V1/inventory/stock-source-links`
+
+**Payload**
+
+```
+{
+   "links" : [
+      {
+         "source_code" : "central",
+         "stock_id" : 4,
+         "priority" : 1
+      },
+      {
+        "source_code" : "east",
+        "stock_id" : 4,
+        "priority" : 2
+      }
+   ]
+}
+```
+
+**Response**
+
+Magento returns empty array.
+
+`[]`
+
+## Delete links between stocks and sources
+
+The `POST /V1/inventory/stock-source-links-delete` endpoint breaks the link between a stock and one or more sources. Magento does not recalculate priority of sources after you delete links.
+
+**Sample Usage**
+
+`POST /V1/inventory/stock-source-links-delete`
+
+**Payload**
+
+```
+{
+   "links" : [
+      {
+        "source_code" : "south",
+        "stock_id" : 4
+      }
+   ]
+}
+```
+
+**Response**
+
+Magento returns empty array.
+
+`[]`
+
+## Search for links
+
+The following call returns the link information for `stock_id = 4`.
+
+See [Search using REST APIs]({{ page.baseurl }}/rest/performing-searches.html) for information about constructing a search query.
+
+**Sample Usage**
+
+`GET /V1/inventory/stock-source-links?searchCriteria[filter_groups][0][filters][0][field]=stock_id&searchCriteria[filter_groups][0][filters][0][value]=4&searchCriteria[filter_groups][0][filters][0][condition_type]=eq`
+
+**Payload**
+
+None
+
+**Response**
+
+{% collapsible Show code sample %}
+``` json
+{
+    "items": [
+        {
+            "stock_id": 4,
+            "source_code": "central",
+            "priority": 1
+        },
+        {
+            "stock_id": 4,
+            "source_code": "east",
+            "priority": 2
+        }
+    ],
+    "search_criteria": {
+        "filter_groups": [
+            {
+                "filters": [
+                    {
+                        "field": "stock_id",
+                        "value": "4",
+                        "condition_type": "eq"
+                    }
+                ]
+            }
+        ]
+    },
+    "total_count": 2
+}
+```
+{% endcollapsible %}
+
+## Get sources assigned to a stock
+
+The `GET /V1/inventory/get-sources-assigned-to-stock-ordered-by-priority/:stock_id` endpoint returns details about each source that is assigned to the specified stock. [Manage sources](manage-sources.html) provides definitions for each attribute returned.
```

Comment:
Link for Manage sources looks a little odd?

Reply:
It was a valid link, but I added the full path, in case the location of the topic changes in the future.